src/main.js

```diff
@@ -1,8 +1,13 @@
+const placeholder = {};
+
+export default placeholder;
+export { default as Label } from './components/Label';
```

Comment:
No need to import/export components from main.js
we should import and use them directly in the components when we need